packages/manager/modules/manager-components/src/index.scss

```diff
@@ -0,0 +1,5 @@
+@import '~bootstrap4/scss/_functions.scss';
```

Comment:
I wonder if this should be scoped to the component (in case we use this in an app that still use bootstrap 3

Reply:
Done